components/Sections/Section.tsx

```diff
@@ -0,0 +1,25 @@
+import { ThemeProvider } from "@/lib/ThemeContext";
+import { ReactNode } from "react";
+import { PremudBreak, PresunBreak } from "../SectionBreak";
+
+interface SectionProps {
+  contentSlot: ReactNode;
+  index: number;
+}
+
+function Section({ contentSlot, index }: SectionProps) {
+  const isMud = index % 2 === 0;
+  const isSun = index % 2 === 1;
+
+  const theme = isMud ? "mud" : "sun";
```

Comment:
is this nessesary? why not `const theme = isSun ? "sun" : "mud";`
then you only need `isSun`